polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -12,11 +12,8 @@ import type {TooltipOverlayProps} from './components';
 import styles from './Tooltip.scss';
 
 export type Width = 'default' | 'wide';
-export type Padding = 'default' | Extract<SpaceScale, '4' | '400'>;
-export type BorderRadius = Extract<
-  BorderRadiusScale,
-  '1' | '2' | '100' | '200'
->;
+export type Padding = 'default' | Extract<SpaceScale, '400'>;
```

Comment:
Removed backwards compatibility for space token here as well as border after confirming that we want to remove backwards compatibility for our migrations in v12 as we're doing migrations.